user-manager/src/integrationTest/java/com/qpay/usermanager/client/AuthenticationClientTest.java

```diff
@@ -0,0 +1,89 @@
+package com.qpay.usermanager.client;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.tomakehurst.wiremock.junit5.WireMockTest;
+import com.qpay.libs.models.UserType;
+import com.qpay.usermanager.model.dto.authuser.UserCredentialsCreation;
+import com.qpay.usermanager.model.dto.authuser.UserCredentialsModification;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.delete;
+import static com.github.tomakehurst.wiremock.client.WireMock.equalToJson;
+import static com.github.tomakehurst.wiremock.client.WireMock.ok;
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.github.tomakehurst.wiremock.client.WireMock.put;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static com.qpay.usermanager.client.AuthenticationClient.AUTHENTICATION_USER_PATH;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest
+@WireMockTest(httpPort = 8186)
+class AuthenticationClientTest {
+
+    @Autowired
+    private AuthenticationClient authenticationClient;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    private static final UserCredentialsCreation USER_CREDENTIALS_CREATION = UserCredentialsCreation.builder()
+            .email("user10mail.com")
+            .password("password")
+            .userType(UserType.CUSTOMER)
+            .build();
+
+    private static final UserCredentialsModification USER_CREDENTIALS_MODIFICATION = UserCredentialsModification.builder()
+            .email("user20mail.com")
+            .password("password1234")
+            .build();
+
+    @Test
+    void should_returnOk_whenSendNewUserSuccessfully() throws JsonProcessingException {
+        // when
+        stubFor(post(urlPathEqualTo(AUTHENTICATION_USER_PATH))
+                .withRequestBody(equalToJson(objectMapper.writeValueAsString(USER_CREDENTIALS_CREATION)))
+                .willReturn(ok()));
+
+        var result = authenticationClient.createUserCredentials(USER_CREDENTIALS_CREATION).getStatusCode();
+
+        // then
+        assertThat(result).isEqualTo(HttpStatus.OK);
+
+    }
```

Comment:
you are not using response entity in the code, except tests. Remove it.
For tests, you can do like this since `stub` validates the request

```
@Test
    void should_returnOk_whenSendNewUserSuccessfully() throws JsonProcessingException {
        // given, then
        stubFor(post(urlPathEqualTo(AUTHENTICATION_USER_PATH))
                .withRequestBody(equalToJson(objectMapper.writeValueAsString(USER_CREDENTIALS_CREATION)))
                .willReturn(ok()));

        
        // when
        authenticationClient.createUserCredentials(USER_CREDENTIALS_CREATION);
    }
```